test/integration/main_test.go

```diff
@@ -528,85 +521,23 @@ func TestPolicyReusableWorkflowCall(t *testing.T) {
 
 // testGetFirstMatchedBQEntryWithRetries queries the DB and get the matched BQEntry.
 // If no matched BQEntry was found, the query will be retried with the specified retry inputs.
-func testGetFirstMatchedBQEntryWithRetries(ctx context.Context, tb testing.TB, bqQuery *bigquery.Query, cfg *config) *bqEntry {
+func testGetFirstMatchedBQEntry(ctx context.Context, tb testing.TB, bqQuery *bigquery.Query, cfg *config) *testutil.BQEntry {
```

Comment:
Why change the naming of the function?

Reply:
Because retry is now handled by `testutil.GetFirstMatchedBQEntryWithRetries`, I think it can be duplicated and confusing for to have `withRetry` in the name.